python/lsst/ts/mtmount/mtmount_csc.py

```diff
@@ -1659,25 +1651,151 @@ async def do_stopTracking(self, data):
         await self.cmd_stopTracking.ack_in_progress(data, timeout=STOP_TIMEOUT)
         await self.send_command(commands.BothAxesStop(), do_lock=False)
 
-    async def _do_applySettingsSet(self, data):
-        self.assert_enabled()
+    async def do_applySettingsSet(self, data):
```

Comment:
Looks like most of the other do_XXX methods have a minimal doc string

```"""Handle the XXX command."""```

so please add that here and to all other do_XXX methods.